### ISSUE TITLE
Suggestion for Refresh Music Player; Implement ability to control from the status bar.

### ISSUE THREAD

**Issue by jubreal** [devendrakushwah/Refresh-Music-Player#8]
#### Components
I am an avid lover of music, which makes me explore a lot of music player apps. And various music applications have their own designs and features that make them the choice to listen to songs at a particular point in time. Some have bulky and heavy features and designs which gives users ability to express themselves to any length they want to, while some music players are very simple which is sometimes preferable based on the mood of the user and the workload on the device. Refresh Music Player is one of those effective lightweight apps, according to the words of the project owner "*The main focus of this project is to create a music player that is optimized for speed and to keep the UI as simple as possible*". I have been using this app for a while now anytime I need a simple music player, but it's missing one important feature which is the ability to control from the status bar.

#### Proposal Description
It has been made clear already that the is a simple music app whose main focus is to create a music player that is **optimized for speed** and to **keep the UI as simple as possible**
So my suggestion is;

Implement the ability to control from status bar.
If this option is implemented, Users will not have to open the app every time they need to perform simple and important music player operations such as Play/Pause option, Next/Previous music option and totally stopping the current music and closing of the app.

This feature is very important in music player app, this will reduce the stress of having to open the whole app before such simple operations can be performed. This app is built for **speed** and this will ensure that by making the access easier and faster through the status bar. And also it will help **keep the UI as simple as possible** because there is a notification bar of the music app which already exist but only serves as notification that the app is currently in use. This can just be modified to accommodate the feature.

#### Mockups / Examples
Below are pictorial illustrations of how the features I recommend can be implemented by the Developer.

This is how the app currently looks like without the features
![WhatsApp Image 2019-04-17 at 4 39 38 PM(1)](https://user-images.githubusercontent.com/37203158/56301925-b7b61a00-6130-11e9-8ab7-69e4d3ea4cba.jpeg)

This is an image showing the proposed features present in the app
![WhatsApp Image 2019-04-17 at 4 39 38 PM](https://user-images.githubusercontent.com/37203158/56301955-c13f8200-6130-11e9-8489-ec82d1ef4ce5.jpeg)

#### Benefits
The benefit of implementing this feature has already been explained in the "proposal" section of this post. In some, if not most cases, users prefer and find it easier to control the music player from the status notification bar. It will be easier and saves a lot of time if the feature is made available to the users and will still be in line with the focus of the project in keeping the UI simple. 
In other words, it saves the user time, makes the app easier to use and more user-friendly.

Thank You.
